### PR TITLE
feat: add richest wallets page

### PR DESCRIPTION
**Pull request by Raysharr**

### CODE REVIEW

File: routes/baseActionsRouter.js [hunk 1]
```diff
@@ -864,6 +864,20 @@ router.get("/about", (req, res) => {
...
...,"amount":4844.02611306,"percentage_of_total":0.14},{"address":"elcash1q0md2zxn7c6cgul0ddlh4pasadkgc89crsjcuze","amount":3747.55641974,"percentage_of_total":0.11},{"address":"EVjQVBuujBnyX8p8i93oDkVGEtzoJrFKFv","amount":3199.99999712,"percentage_of_total":0.09},{"address":"EUsZriyUE3wbNkpSrTncXMdnPhMRkpj8vV","amount":2832.66169757,"percentage_of_total":0.08},{"address":"elcash1qrr0yn22yukxpf26esr3msy6m2wyef3wkn5q9fm","amount":2596.4031261,"percentage_of_total":0.07},{"address":"ENE6tLMjWomk5o92chneH9be1Xh9ozzzsw","amount":2379.99999618,"percentage_of_total":0.07},{"address":"elcash1q3u4ynvlv0chv9lpgc6jyutsk7rjl23k0sfdgrr","amount":1860.53656953,"percentage_of_total":0.05},{"address":"Eb73uRS6irH12mUjKMG9F2K7bG1kyr6Fmp","amount":1766.0400939,"percentage_of_total":0.05},{"address":"EPuTrWMpWW8dYUib5h6fFMM2S2dWQBCTCN","amount":1649.24457282,"percentage_of_total":0.05},{"address":"elcash1qxlvmfyn46d42lzyrw3l8emnr83dj0fly3280sa","amount":1604.70478101,"percentage_of_total":0.05},{"address":"elcash1qthnl6apf44qmz59p2l0p6w0ylttyyteqnzg7cz","amount":1576.94577274,"percentage_of_total":0.04},{"address":"EUXtb3rpsCZA2W7WXPiJVAM4T6dRw6mT8u","amount":1513.71981932,"percentage_of_total":0.04}];
```
Comment (krecik) [inline routes/baseActionsRouter.js:868]: Please remove this comments.

**Review by krecik — CHANGES_REQUESTED** [review]

File: routes/baseActionsRouter.js (hunk 1 above)
Comment (Raysharr) [reply routes/baseActionsRouter.js:868]: sure

File: routes/baseActionsRouter.js [hunk 2]
```diff
@@ -864,6 +864,16 @@ router.get("/about", (req, res) => {
 	res.render("about");
 });
 
+router.get("/richest-wallets", (req, res) => {
+	utils.getRichestWallets().then(function(results) {
+		global.richestWallets = results;
+
+		res.render("richest-wallets");
+
+		next();
```
Comment (krecik) [inline routes/baseActionsRouter.js:873]: This will not work, in function definition you don't have next function. Remove this call.

**Review by krecik — CHANGES_REQUESTED** [review]

File: routes/baseActionsRouter.js (hunk 2 above)
Comment (Raysharr) [reply routes/baseActionsRouter.js]: done

**Review by picq — APPROVED** [review]

**Review by krecik — APPROVED** [review]

**Pull request merged**